.github/workflows/tests.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       run: ./.github/bin/install-zig.sh
 
     - name: Test exercises
-      run: ./_test/run-tests.sh
+      run: ./bin/run-tests.sh
```

Comment:
This makes it be more in line with other tracks. It also increases visibility I feel.